lib/carbon/database.py

```diff
@@ -54,6 +59,26 @@ def validateArchiveList(self, archiveList):
     "Validate that the database can handle the given archiveList."
     pass
 
+  def tag(self, metric, httpRequest=None):
```

Comment:
I don't think this function needs to take httpRequest in argument (it only depends on reactor, which is global)